ldes-server-port-admin-rest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/AdminEventStreamsRestControllerTest.java

```diff
@@ -100,7 +100,7 @@ void when_StreamsPresent_then_StreamsAreReturned() throws Exception {
 
 			mockMvc.perform(get("/admin/api/v1/eventstreams").accept(contentTypeTurtle))
 					.andExpect(status().isOk())
-					.andExpect(IsIsomorphic.with(expectedEventStreamsModel));
+					.andExpect(IsIsomorphicResult.with(expectedEventStreamsModel));
```

Comment:
this was a rename? Since I find this now less readable?

Reply:
Yes because we I added a second implementation so we had 'IsIsomorphicResult' and 'IsIsomorphicArgument'. However I agree with your comment and changed it back to one class that now implements the two interfaces `implements ResultMatcher, ArgumentMatcher<Model>`